docs/deploying.md

```diff
@@ -79,46 +77,45 @@ This document will cover writing the `YAML` document for an
 with `oc` as the last step.
 
 Start the `YAML` document in an `openstack-edpm.yaml` file and give the
-dataplane a name.
+OpenStackDataPlaneNodeSet a name.
 
     apiVersion: dataplane.openstack.org/v1beta1
     kind: OpenStackDataPlaneNodeSet
     metadata:
       name: openstack-edpm
 
-Begin writing the dataplane spec. Initially, a `deployStrategy` field will be
-added to the spec that contains `deploy: false`. This allows for creating
-the dataplane resources without triggering an Ansible execution immediately.
+An OpenStackDataPlaneNodeSet represents a set of nodes that are configured in a
+similar way. Nodes that are deployed with common configurations and that share
+a set of common ansible variables can be deployed using the same
+OpenStackDataPlaneNodeSet.
 
-    apiVersion: dataplane.openstack.org/v1beta1
-    kind: OpenStackDataPlaneNodeSet
-    metadata:
-      name: openstack-edpm
-    spec:
-      deployStrategy:
-          deploy: false
-
-Add roles to the dataplane. This example uses a single role, but any number
-could be added. A single role called `dataplane-role` is added for this
-example. Under the role, a `preProvisioned` field is set to `True` since these
-nodes are preprovisioned. A `nodeTemplate` field is also started that contains
-the fields that will have their values inherited by each node in the role. See
-[Inheritance](inheritance.md) for more details about how role and node
-inheritance works. Within `nodeTemplate`, the fields shown are documented
-inline in the example.
+Consider using different OpenStackDataPlaneNodeSets to logically group nodes in
+a way that makes sense. Differences between nodes are likely to include
+configurations due to hardware, location, or networking.  As differences grow,
+use different OpenStackDataPlaneNodeSets to manage similarly configure nodes
```

Comment:
Please use "configured" not "configure". 

"to manage similarly configured nodes"

"more easily" can be removed

Reply:
Done.